proto/types.proto

```diff
@@ -11,3 +11,15 @@ message ExtraData {
 	bytes JsonSpec = 1;
 	google.protobuf.Any RuncOptions = 2;
 }
+
+// Message to specify network config for a Firecracker VM
+message FirecrackerNetworkInterface {
```

Comment:
Should we include addition IPv4 configuration fields in this message? In your example, you provide IPv4 configuration to the kernel via a hardcoded commandline (`ip=...`). If we want to provide this information dynamically, this message seems to be the place to do it. If you'd rather save that for a later PR, though, that's fine.

Reply:
Yeah, my intention is to do that in a later PR as we'll be adding additional fields anyway as per #76. We can then get rid of that manual override in the example as well.